src/JPEGView/Config/JPEGView_ru.ini

```diff
@@ -52,7 +52,14 @@ FilesProcessedByWIC=*.wdp;*.hdp;*.jxr
 ; RAW-����� �������� �����, � ������� ������� ������ ��������������� ���
 ; ������������� ���������� JPEG-�����������. ���������� ����� ���������� JPEG
 ; ����������� ������� �������, ��� ������������� RAW-������ � ������� WIC.
-FileEndingsRAW=*.pef;*.dng;*.crw;*.nef;*.cr2;*.mrw;*.rw2;*.orf;*.x3f;*.arw;*.kdc;*.nrw;*.dcr;*.sr2;*.raf
+FileEndingsRAW=*.pef;*.dng;*.crw;*.nef;*.cr2;*.mrw;*.rw2;*.orf;*.x3f;*.arw;*.kdc;*.nrw;*.dcr;*.sr2;*.raf;*.kc2;*.erf;*.3fr;*.raw;*.mef;*.mos;*.mdc;*.cr3
+
+; ��������� ��� �������� RAW-������ �� ������ FileEndingsRAW

```

Comment:
Wrong Cyrillic letters in the first line. Should be fixed like this:

```diff
- ; Џоведение длЯ открытиЯ RAW-файлов из списка FileEndingsRAW
+ ; Поведение для открытия RAW-файлов из списка FileEndingsRAW
```

Probably it was an issue with `windows-1251` encoding which used back in the days for languages that use Cyrillic.
Maybe it's time to finally convert these files to Unicode?